drivers/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+add_subdirectory_ifdef(CONFIG_SENSOR sensor)
```

Comment:
The build system is a maze of twisty little passages. Comments in every file explaining their purpose -- even just tweet-length ones -- would go a long way towards demystifying.